ui/src/app/components/interactive.tsx

```diff
@@ -0,0 +1,100 @@
+import * as Utils from 'app/utils';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+
+const {useState, forwardRef} = React;
+const allowedHoverVariables = ['backgroundColor', 'border', 'color', 'boxShadow', 'opacity', 'textDecoration'];
+const pointerTags = ['button', 'area', 'a', 'select'];
+const pointerTypes = ['radio', 'checkbox', 'submit', 'button'];
+
+interface InteractiveProps {
+  className?: string;
+  type?: string;
+  role?: string;
+  onClick?: () => any;
+  onKeyDown?: () => any;
+  onMouseDown?: Function;
+  onBlur?: Function;
+  disabled?: boolean;
+  children?: React.ReactElement | React.ReactElement[];
+  tabIndex?: number;
+  hover?: React.CSSProperties;
+  style?: React.CSSProperties;
+  tagName: keyof JSX.IntrinsicElements;
+}
+
+export const Interactive: React.ForwardRefExoticComponent<InteractiveProps> = forwardRef(({
+  className = '',
+  tagName: TagName = 'div',
+  type,
+  role,
+  onClick,
+  onKeyDown,
+  onMouseDown,
+  onBlur,
+  disabled,
+  children,
+  tabIndex,
+  hover = {},
+  style = {},
+  ...props}: InteractiveProps, ref) => {
+  const [outline, setOutline] = useState(undefined);
+  const { cursor } = style;
+
+  const computedCursor = Utils.cond(
+    [!!cursor, () => cursor],
+    [disabled, () => undefined],
```

Comment:
This looks like a change in behavior from the existing playground-mode.  It has 
`const disabledStyle = {cursor: 'not-allowed', fill: colorWithWhiteness(colors.dark, 0.6)};` which is something the e2e tests depend on to detect if something is disabled.

Reply:
I will take a look at this to see what can be done to address this.

Reply:
I added the cursor styling on disabled. Puppeteer tests passed.
I did not see any test functions checking the fill or color for disabled.
Thanks for catching this!